src/handleInput.js

```diff
@@ -34,7 +35,8 @@ export default async function handleInput(
   if (
     event.input.length >= 3 &&
     !event.input.startsWith(REASON_PREFIX) &&
-    !event.input.startsWith(DOWNVOTE_PREFIX)
+    !event.input.startsWith(DOWNVOTE_PREFIX) &&
+    state !== 'ASKING_ARTICLE_SOURCE'
```

Comment:
This is because currently the newly added options have more than 3 characters.

Reply:
we won't need this after changing to use 1~4 in buttons.

Reply:
Indeed. Fixed.